addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"account_account_template_4220","Long-Term Liabilities under Loan Agreements",4220,"account.data_account_type_non_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_1_2","False"
+"account_account_template_4221","Other Obligations",4221,"account.data_account_type_non_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_1_2","False"
+"account_account_template_4230","Prepayments from Customers",4230,"account.data_account_type_non_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_1_3","False"
+"account_account_template_4231","Prepayments from Service Receivers",4231,"account.data_account_type_non_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_1_3","False"
+"account_account_template_424","Debts to Suppliers",424,"account.data_account_type_non_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_1_4","False"
+"account_account_template_428","Other Payables and Long-Term Liabilities",428,"account.data_account_type_non_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_1_8","False"
+"account_account_template_4401","Part of Leasing (Financial Rent) or Other Obligations in The Current Year",4401,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_1","False"
+"account_account_template_4403","Part of Other Long-Term Debts in The Current Year",4403,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_1","False"
+"account_account_template_4410","Liabilities under Short-Term Loan Agreements",4410,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_2","False"
+"account_account_template_4411","Part of Debts to Credit Institutions in The Current Year",4411,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_2","False"
+"account_account_template_4413","Other Obligations (Executables and etc.)",4413,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_2","False"
+"account_account_template_4420","Prepayments from Customers",4420,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_3","False"
+"account_account_template_4421","Prepayments from Service Receivers",4421,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_3","False"
```

Comment:
Accounts `4420` and `4421` could (maybe even should) have type **Prepayments** which is even in their name.

```suggestion
"account_account_template_4421","Prepayments from Service Receivers",4421,"account.data_account_type_prepayments","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_3","False"
```